catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsTest.java

```diff
@@ -402,6 +406,37 @@ public void testGetResourceInfoUriNotBlank() throws Exception {
         verify(resourceRequestMock).getAttributeName();
     }
 
+    /**
+     * Ensure the uri is preserved in resource requests that are done by resource-uri
+     * If the uri is not preserved exactly then retrieval can fail
+     */
+    @Test
+    public void testGetResourceInfoPreservesFragment() throws Exception {
```

Comment:
❗️ While you're in this area, I see that we have clashing class names. `ResourceOperationsTest.java` and `ResourceOperationSpec.groovy` files both define the class `ResourceOperationsTest` (the latter was incorrectly renamed, it seems). That's impacting our test coverage. We need to resolve.

I would personally argue that the junit test's contents should be translated/transferred to the Spock test as that is where the majority of the coverage lives (and originally lived) for `ResourceOperations`, but that is beyond the scope of this ticket.